doc/rbd/rbd-encryption.rst

```diff
@@ -240,6 +240,17 @@ The same applies to creating a formatted clone of an unformatted
 (plaintext) image since an unformatted image does not have a header at
 all.
 
+To map a formatted clone of an encrypted parent image, provide the
+passphrase file of the clone and its encrypted parent image. The order in
+which the passphrase files should be provided is based on the image
+hierarchy, starting with that of the clone image, then its parent and so on.
+
+.. prompt:: bash $
+
+    rbd device map -t nbd -o encryption-passphrase-file=clone-passphrase.bin -o encryption-passphrase-file=passphrase.bin mypool/myclone
+
+
+
```

Comment:
Are that many blank lines needed for better rendering?  If not, let's leave just one.